tests/unit/io/test_eventletreactor.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest # noqa
+
+from tests.unit.io.utils import submit_and_wait_for_completion, TimerCallback
+from tests import is_eventlet_monkey_patched
+import time
+
+try:
+    from cassandra.io.eventletreactor import EventletConnection
+except ImportError:
+    EventletConnection = None  # noqa
+
+
+class EventletTimerTest(unittest.TestCase):
```

Comment:
Looks like this is a legacy issue. I'll change it over to use the same monkey patch code I use in the libev timer tests. That should check for both kinds of monkey patching.